src/python/pants/backend/python/macros/python_artifact.py

```diff
@@ -21,6 +45,12 @@ def __init__(self, **kwargs):
                 f"{type(name)}."
             )
 
+        # coerce entry points from dict of string -> dict of string -> string to
+        # dict of string -> list of string
```

Comment:
May be easier to read this as `dict[str, dict[str, str]]` -> `dict[str, list[str]]`.